agoradatatools/etl/extract.py

```diff
@@ -85,4 +87,16 @@ def read_feather_into_df(feather_path: str):
               + str(feather_path) + " matches the file extension.")
         sys.exit(errno.EBADF)
 
-    return pd.read_feather(feather_path)
\ No newline at end of file
+    return pd.read_feather(feather_path)
+
+def read_json_into_df(json_path: str):
+    """
+    Reads a json file from synapse into a dataframe
+    """
+
+    if json_path.split(".")[-1] != "json":
+        print("Please make sure the format parameter in the configuration for "
+              + str(json_path) + " matches the file extension.")
```

Comment:
Minor suggestion

```suggestion
              f"{str(json_path)} matches the file extension.")
```

Reply:
I like f-strings far better.  Thanks for the suggestion.